### DISCUSSION TITLE
How to ignore page hosted on another site. 301 redirect: blog.domain.com >> domain.com/blog

### DISCUSSION THREAD

**Discussion by YoungiiJC** [vercel/next.js#23715] [Help]
Hey there. My nextjs app is on domain.com. I have an external blog that lives on blog.domain.com. I'd like to route blog.domain.com to domain.com/blog with a 301. Can nextjs handle this sort of thing?

**Comment by YoungiiJC** [comment]
Ping.

**Comment by akellbl4** [comment]
If I understand correctly you want to redirect from blog.domain.com to domain.com/blog and it could be solved only with a redirect on the subdomain.
What is on that subdomain serving content? You need to make a redirect there. If you don't know what is there and don't have access to the configuration you can solve it with HTML in the head:
```
<meta http-equiv="refresh" content="0;url=https://example.com/blog" />
```
